src/clang.rs

```diff
@@ -809,27 +802,65 @@ impl Comment {
         }
     }
 
-    /// Given that this comment is an HTML start tag, get the number of HTML
-    /// attributes it has.
-    pub fn get_num_tag_attrs(&self) -> c_uint {
-        unsafe {
-            clang_HTMLStartTag_getNumAttrs(self.x)
+    /// Given that this comment is an HTML start tag, get its attributes.
+    pub fn get_tag_attrs(&self) -> CommentAttributesIntoIterator {
+        CommentAttributesIntoIterator {
+            x: self.x,
+            length: unsafe { clang_HTMLStartTag_getNumAttrs(self.x) },
+            index: 0
         }
     }
+}
 
-    /// Given that this comment is an HTML start tag, get the `idx`th
-    /// attribute's name.
-    pub fn get_tag_attr_name(&self, idx: c_uint) -> String {
-        unsafe {
-            String_ { x: clang_HTMLStartTag_getAttrName(self.x, idx) }.to_string()
+/// An iterator for a comment's children
+pub struct CommentChildrenIntoIterator {
```

Comment:
nit: Probably `CommentChildrenIterator`? `IntoIterator` usually means that it "consumes" the value, which doesn't make a lot of sense in this case.